src/sass/style.scss

```diff
@@ -0,0 +1,36 @@
+@import 'bootstrap';
+@import url('https://fonts.googleapis.com/css2?family=IBM+Plex+Sans+Condensed:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;1,100;1,200;1,300;1,400;1,500;1,600;1,700&family=Tourney:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
+
+// font-family: 'Tourney', cursive;
+// font-family: 'IBM Plex Sans Condensed', sans-serif;
```

Comment:
- [ ] Still, I can see the commented code. Please remove that to make the codebase more cleaner.